libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using prompts = Microsoft.Bot.Builder.Prompts;
```

Comment:
Why the import like this? Just do a standard using and avoid the "prompts." everywhere.